README.rst

```diff
@@ -36,17 +36,13 @@ This is easily achieved by downloading
 or individual libraries can be installed using
 `circup <https://github.com/adafruit/circup>`_.
 
-.. todo:: Describe the Adafruit product this library works with. For PCBs, you can also add the
-image from the assets folder in the PCB's GitHub repo.
+* `Waveshare 7.3" F <https://www.waveshare.com/7.3inch-e-paper-hat-f.htm>`_
 
-`Purchase one from the Adafruit shop <http://www.adafruit.com/products/>`_
 Installing from PyPI
 =====================
 .. note:: This library is not available on PyPI yet. Install documentation is included
    as a standard element. Stay tuned for PyPI availability!
```

Comment:
Since Thonny looks for PyPI, everything is being added there.  The release workflow will do it and now there's no `setup.py.disabled` file to prevent it.  Thoughts?

Reply:
I think that's correct it looks like it will get pushed to pypi once a release is made if everything works out. 

This note is just leftover from the cookiecutter, I recall seeing it before. Maybe there is a case in there where we could tweak the default to not include this message for adafruit libraries.

I tried to push the change here to remove it, but don't have access (or my ide/git integration is messed up). 

I found another small change needed for this repo and the other 7 color eink one.

I'll merge this and then remove the note in a followup with that other change.

Reply:
the problem pushing does seem to be something wonky on my side. I had the same problem pushing to my own fork but think It's straightened out for now.